lib/puppet/type/posix_acl.rb

```diff
@@ -1,5 +1,6 @@
 require 'set'
 require 'pathname'
+require 'English'
```

Comment:
why is this needed?

Reply:
It gives you `$LAST_MATCH_INFO`

Reply:
And really the only reason I switched from $~ to $LAST_MATCH_INFO because rubocop yelled at me and questioned my lineage.  ^_^